src/components/ListView.jsx

```diff
@@ -24,18 +25,25 @@ function ListView() {
       //Used to remove the snapshot listener when the component is unmounted/unsubscribed
       unsubscribe();
     };
-  }, []);
+  }, [localToken, navigate]);
 
   return (
-    <div>
-      Your Shopping List:
-      <ul>
-        {items.map((item, idx) => (
-          <li
-            key={idx}
-          >{` Name: ${item.itemName}  Frequency: ${item.frequency}`}</li>
-        ))}
-      </ul>
+    <div className="m-20">
+      {!items.length ? (
+        <Welcome items={items} />
+      ) : (
+        <div>
+          <h1 className="text-center text-2xl my-12">Your Shopping List:</h1>
+          <ul className="grid grid-cols-2 justify-around">
```

Comment:
I like that you're judicious which Tailwind classes are used instead of copying overly complex template as a starting point